.github/workflows/ci.yml

```diff
@@ -75,6 +75,9 @@ jobs:
       - name: Run rails tests
         run: bundle exec rails test
 
+#      - name: Run vitest tests
```

Comment:
why is this commented out?

Reply:
It fails because there aren't any tests yet. I uncommented it in the PR with the tests.